### ISSUE TITLE
Is it possible to init a Python project?

### ISSUE THREAD

**Issue by ssmirnov777** [apex/apex#694]
"apex init" creates  a nodejs handler by default. Is it possible to start a python handler right away?

**Comment by polds** [comment]
The init process just drops a sample NodeJS function, this doesn't bind the entire project to Node, you could potentially mix languages too. Just delete the NodeJS function and start adding your Python functions and you're good to go.

**Comment by ssmirnov777** [comment]
Thank you for the answer. It is not easy to find a step-by-step instruction how to migrate just created nodejs project to the python one.

**Comment by tj** [comment]
Yeah just node for now sorry :( it would be quite a bit of work to have one per runtime. Hopefully the examples will help https://github.com/apex/apex/tree/master/_examples

**Comment by ssmirnov777** [comment]
This is my version of "step-by-step":
1. remove index.js
2. add lambda_function.py with the following minimal code:
```
def lambda_handler(event, context):
    return "hello apex!"
```
3. add the following  params to the end of the project.json:
```
  "runtime": "python",
  "handler": "lambda_function.lambda_handler"
```
4. deploy and test

===
both params are mandatory in my case. However, I did not see them in the the python's example in the repo. Is it another way to point the actual runtime?

**Comment by tj** [comment]
Yep, Apex has some [inference](https://github.com/apex/apex/blob/master/plugins/inference/inference.go#L18) and a [default handler](https://github.com/apex/apex/blob/master/plugins/python/python.go#L30).

**Comment by ssmirnov777** [comment]
Great! My instruction became twice shorter. It might be nice to mention those predefined files names somewhere in the doc.

**Comment by tj** [comment]
agreed! I'll try and remember to get that in there

**Comment by tchia04** [comment]
I wished I had read this issue earlier. It would have saved me couple hours trying to initialize a python project/function.
